spectractor/fit/fit_spectrum.py

```diff
@@ -78,36 +83,41 @@ def __init__(self, file_name, atmgrid_file_name="", nwalkers=18, nsteps=1000, bu
         self.data = self.spectrum.data
         self.err = self.spectrum.err
         self.data_cov = self.spectrum.cov_matrix
+        self.fit_angstrom_exponent = fit_angstrom_exponent
         self.A1 = 1.0
         self.A2 = 0
         self.ozone = 400.
         self.pwv = 5
         self.aerosols = 0.01
+        self.angstrom_exponent_log10 = -2
```

Comment:
Ah, having read on is this -2 because of the value of 0.0192 later, and this is close to the log10 of that? In which case, why -2, rather than an exact -1.7166? (or even `math.log10(0.0192)`)?

Reply:
Well... because either you fit this parameter with fit_angstrom_exponent and starting from -2 is a good choice, either we don't fit it and then it's value has no impact.

Reply:
Got it, thanks.